transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/TransactionOutbox.java

```diff
@@ -88,6 +88,8 @@ static TransactionOutboxBuilder builder() {
   @SuppressWarnings("UnusedReturnValue")
```

Comment:
Javadoc needs updating to reflect that grouped records are ignored and these should be handled by `flushOrdered`

Reply:
Yeah, they will be added (I've already written some of them but haven't pushed yet)

Reply:
@amseager Would you please be able to push those changes?

Reply:
@amseager Would you be able to push those changes?